firebase-installations/src/main/AndroidManifest.xml

```diff
@@ -11,4 +11,6 @@
                 android:value="com.google.firebase.components.ComponentRegistrar" />
         </service>
     </application>
+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
you need the internet permission as well (a comment for the last cl, lol)

Reply:
Done. Thanks :)